aws/resource_aws_dms_endpoint.go

```diff
@@ -306,6 +306,33 @@ func resourceAwsDmsEndpoint() *schema.Resource {
 							Optional: true,
 							Default:  false,
 						},
+						"data_format": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							Default:      dms.DataFormatValueCsv,
+							ValidateFunc: validation.StringInSlice(dms.DataFormatValue_Values(), false),
+						},
+						"parquet_timestamp_in_millisecond": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Default:  false,
+						},
+						"parquet_version": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							Default:      dms.ParquetVersionValueParquet10,
+							ValidateFunc: validation.StringInSlice(dms.ParquetVersionValue_Values(), false),
+						},
+						"encryption_mode": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							Default:      dms.EncryptionModeValueSseS3,
+							ValidateFunc: validation.StringInSlice(dms.EncryptionModeValue_Values(), false),
```

Comment:
Hi @sbrandtb . Thanks for the above changes! This one though, you should keep the string values as you had before ("SSE_S3" and "SSE_KMS") since the const values provided by the AWS SDK Go are in a different case/use hyphens and we'll get a perpetual diff after resource creation

Reply:
also, looks like even the API validation fails with the sdk constants 🙃 
```
        Error: Error creating DMS endpoint: InvalidParameterCombinationException: Only SSE_S3 or SSE_KMS encryption mode supported. 'sse-s3' is not supported
```

Reply:
That's unfortunate :/ changed